### PR TITLE
fix: display tag if question is answered

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx [hunk 1]
```diff
@@ -8,29 +8,47 @@ describe('CoreJsIviewsModal', () => {
...
+          courseFullName: 'TEST COURSE',
+          courseName: 'TEST COURSE',
+          locationName: null,
           interviews: [
             {
+              score: 9,
+              comment: 'dasbfjvf dfjvdxfjbv fdlxcbckbgcfgbgfnb n  mgfnm',
+              interviewer: {
+                name: 'anik188',
+                githubId: 'anik188',
```
Comment (mikhama) [inline client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx:20]: please use some fake github id

File: client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx [hunk 2]
```diff
@@ -8,29 +8,47 @@ describe('CoreJsIviewsModal', () => {
...
-          courseFullName: 'rs-2019',
-          courseName: 'rs-2019',
-          locationName: 'minsk',
+          courseFullName: 'TEST COURSE',
+          courseName: 'TEST COURSE',
+          locationName: null,
           interviews: [
             {
+              score: 9,
+              comment: 'dasbfjvf dfjvdxfjbv fdlxcbckbgcfgbgfnb n  mgfnm',
```
Comment (mikhama) [inline client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx:17]: Please write something human readable

File: client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx]: done

File: client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/components/Profile/__test__/CoreJsIviewsModal.test.tsx]: done

**Review by mikhama — APPROVED** [review]

**Pull request merged**